src/sage/combinat/designs/designs_pyx.pyx

```diff
@@ -15,6 +15,199 @@ from cysignals.memory cimport sig_malloc, sig_calloc, sig_realloc, sig_free
 
 from sage.misc.unknown import Unknown
 
+#Current Version
+#***********************************************************#
+
+def is_covering_array(array, strength=None, levels=None, verbose=False, parameters=False):
+    r"""
+    Check if the input is a covering array with given strength.
+
+    - ``array`` -- the Covering Array to be tested.
```

Comment:
The list of arguments should be preceded with `INPUT:`, see [sage developer guide, docstring section](https://doc.sagemath.org/html/en/developer/coding_basics.html#the-docstring-of-a-function-content).